java/maven/README.md

```diff
@@ -3,13 +3,13 @@
 ## Building
 
 ```bash
-pack build applications/maven
+pack build applications/maven --builder paketobuildpacks/builder-jammy-base -e BP_JVM_VERSION=17
```

Comment:
I think this is a fair criticism of all of the examples. None of them are specifying a builder and there's an implied assumption that you have set a default builder with pack. If you don't set a default builder, our sample commands will fail, same as if you have the default set to some other builder or a buildpackless builder.

It seems like we could solve this by a.) adding `--builder` flags to all of the arguments or b.) making an actual prerequisites section somewhere (probably the paketo.io/docs site) and linking to that from the READMEs. I'm leaning towards b.) but I don't have a strong preference, I'll leave that up to you.

I will say that I'd like to see a.) or b.) consistently applied across the Java samples though. Would you be able to modify this PR to update all of the Java samples?